File: 백준/Bronze/4998. 저금/저금.py
import sys
input = lambda: sys.stdin.readline().rstrip()


def main():
    while 1:
        inp = input()
        if inp == '':
            break
        N, B, M = map(float, inp.split())
        year = 0
        while 1:
            if N > M:
                break
            N += N*B/100
            year += 1
        print(year)


if __name__ == '__main__':
    main()

File: 백준/Bronze/4998. 저금/저금.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        
        while (true) {
            String inp = br.readLine();
            if (inp == null || inp.trim().isEmpty()) {
                break;
            }
            
            String[] parts = inp.split(" ");
            double N = Double.parseDouble(parts[0]);
            double B = Double.parseDouble(parts[1]);
            double M = Double.parseDouble(parts[2]);
            
            int year = 0;
            while (N <= M) {
                N += N * B / 100;
                year++;
            }
            System.out.println(year);
        }
        br.close();
    }
}